recipe/meta.yaml

```diff
@@ -16,10 +16,10 @@ build:
 
 requirements:
   host:
-    - python >=3.7
```

Comment:
Minumum Python is [`>=3.8`](https://github.com/keras-team/autokeras/blob/master/pyproject.toml)